### DISCUSSION TITLE
Some steps to deploy Sails Js apps?

### DISCUSSION THREAD

**Discussion by nicolascalev** [vercel/vercel#4639] [Help] [answered]
Hi! So Sails Js is a Node Js framework but I don't know why it didn't work for me.
I got some erros while trying but I'm definetly interested. I might try uploading a Docker image 🤨

**Comment by mcsdevv** [comment, marked as answer]
Hello @guillennicolas, thanks for raising this!

SailsJS won't work as expected because it's a Node.js framework designed for a **running server**.

The Vercel platform is serverless and designed for static sites and [Serverless Functions](/docs/v2/serverless-functions/introduction).

**Reply by nicolascalev** [reply]
Oh ok thank you! I hadn't used it since it became Vercel. I watched a course about deploying with now and so I thought it was
possible. My bad

Thanks tho! 😉

**Reply by mcsdevv** [reply]
Hey @nicolascalev, not to worry! Can you let me know which course it was?

**Reply by nicolascalev** [reply]
> Hey @nicolascalev, not to worry! Can you let me know which course it was?

https://platzi.com/clases/deploy-now/
Not very useful given that docker images have been deprecated too, but there it is 😃

**Reply by mcsdevv** [reply]
Thanks for letting me know!
